scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
But generators are not run directly by the user command-line. They are run by CMakeLists as part of a luajit command. Rather than second-guessing argument lists, providing `make gen_xx` would be a better manual invoke point. If there is `generators/gen_xx.lua` and `generators/lib/xx.lua`, an extra `bin/` will only be a many a wasted tab completion for no gain.